pandas/util/testing.py

```diff
@@ -1326,12 +1326,13 @@ def assert_frame_equal(left, right, check_dtype=True,
         Second DataFrame to compare.
     check_dtype : bool, default True
         Whether to check the DataFrame dtype is identical.
-    check_index_type : {'equiv'} or bool, default 'equiv'
+    check_index_type : bool / string {'equiv'}, default 'equiv'
```

Comment:
I think this and everything below it in this diff is from a botched rebase. Trying to fix it...